media/mp4.ksy

```diff
@@ -0,0 +1,1450 @@
+# Argon Design Ltd.
+# Copyright (c) 2019 Argon Design. All rights reserved
```

Comment:
"Copyright + All rights reserved" seems to contradict `meta/license`. Can you clarify how you want to license this work?